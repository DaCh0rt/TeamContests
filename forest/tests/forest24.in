572 674
-8866 -8462
6893 -5759
1409 -9131
5035 1963
6710 -6976
2794 2296
6966 7770
-1464 2269
8115 263
1657 -1339
8463 -2669
1338 -5579
-5518 -9644
7304 9506
6286 -4299
6231 -3276
3281 -4734
6223 57
-7170 -9995
8471 6293
799 4560
-9669 -9737
-4884 9681
6613 -8459
7857 9374
-5444 -899
6279 5566
8350 -1674
7919 2678
3220 4432
-9896 6307
7472 1273
-3691 -8839
-4105 6564
2232 -7997
4072 -565
-1290 5382
-5387 -6386
3847 -4263
8207 -5033
-1223 7522
-6726 6775
-5362 -6639
-1671 -4301
5761 4615
-2284 -7126
3261 -483
2170 -131
-8178 -6971
-6940 -6428
-8022 2099
5486 -7628
737 6148
-828 -7337
4760 2980
-3671 3399
-1764 8886
7766 -1307
9813 -2535
5910 2999
5518 6746
8201 -3208
-4445 6234
2737 -9716
-9773 -4590
-595 2340
-327 -9397
-3909 -6109
5209 -1975
-1536 2332
9390 -1498
-5140 -4094
8245 -6999
4882 -8052
9697 -8866
-6018 -5120
-1362 9537
-1690 9516
-8858 -533
-3191 48
-6867 -2138
5836 8584
6232 4271
6952 3941
1283 1744
-3994 7805
-3067 9126
7807 3126
2071 517
-1940 -8206
-6785 -2767
1071 6407
2186 3294
-5465 -4070
-3867 -8752
4242 8226
3558 -3187
4839 2416
1935 1866
-5966 -959
-6100 4883
-3228 -4816
-707 -6454
-4881 428
-3402 2300
-4489 9942
-3599 -6953
-8443 -5195
5859 -3789
-3057 2391
-5324 3560
-1770 4820
6596 6833
2074 9740
-3405 1267
7045 -7417
1900 5583
-4194 600
6898 -5187
3664 -3402
-4351 -2708
7808 2434
-2168 1028
-4323 9277
-8231 6016
-5956 -3567
-546 -8937
4845 -9836
7042 -7461
5550 7839
7107 69
1397 9980
1021 -1981
644 37
-6104 7660
2283 -9203
-4284 -3249
8 9561
6815 6790
1811 6086
2735 -8504
3196 -3097
7322 6814
430 -2774
2768 -9800
2140 2555
6713 150
-1321 3570
2276 -3487
-2129 -3642
-3982 690
7433 644
-7713 2831
2671 -8472
-2526 6684
4417 2398
5266 -2612
-8652 3932
7700 3033
-6144 1287
-6600 -7016
-3826 -7599
-1432 7663
-7616 -9781
-8187 3084
-8041 3566
-2876 -4354
1415 8579
-3467 6618
-6748 2056
-9194 -6813
918 -1251
713 628
-3606 -1075
2869 1414
-5442 -424
-5241 2536
-6474 -1732
8838 6259
-3348 1565
-7756 -6436
-3781 -2001
-7857 -2727
-7294 4841
-7425 2003
6436 -4772
9095 2398
5197 -4005
-8205 4258
-6348 -2219
-5565 -2467
8998 -8370
-7885 4808
6649 2920
1145 8343
-8698 -5608
7920 -7979
-869 -176
-5280 -1025
-9576 -2152
208 4824
6437 -5477
2852 4639
-8819 -5361
8155 -5871
-9882 -6984
4260 4656
-5233 9445
-6367 -6857
-4558 -2854
6987 -490
296 4617
-567 -4914
5825 -3228
-8306 1409
7747 -5219
4334 6765
-4441 -7932
2448 94
4027 6868
1063 -8927
2239 4898
6635 -1968
2680 6883
1901 8275
-5379 462
-6796 -4898
5523 -7423
6591 7459
6496 -7650
4068 -5353
353 6424
-5411 427
-6838 5130
7483 1475
7086 -5503
8291 -7268
3906 -2781
-6957 1216
3174 -7615
-6908 7272
7866 -9573
2571 -1654
6495 345
8853 5527
-1583 6657
-2217 -5495
-8428 -4299
1731 896
-4230 9896
3034 2782
-920 8560
3836 328
2757 3963
8043 7230
1413 881
9986 6199
3651 -1020
-8774 -6763
1778 -5298
3214 -4706
7174 3875
-4303 7851
4523 -5840
1289 -1461
3534 -8376
6149 3010
-2806 -9212
7890 4804
-605 8536
-3462 -4619
3305 -1570
-8055 -2642
-245 6649
-799 3946
6374 -4240
9398 -5257
8905 7366
-1104 -564
3499 -8263
-5263 7217
-8512 3059
-3886 -5823
6142 7687
2350 -3410
-9236 5341
4415 408
8877 -4558
4744 -9
5182 -9270
-4568 -2185
-8894 8495
-3449 -139
6515 -3948
-663 9626
-2184 -6313
-7875 -7127
1608 6836
-1124 2690
1853 3619
4700 8153
7556 4556
4317 7036
7417 1915
-4369 -1028
4027 5750
1402 -837
4392 4537
4153 5034
-8810 2628
-741 9541
-7312 1779
133 2099
-4161 -7594
6900 7463
-5794 1870
9022 -6244
9664 2600
6733 -3453
2352 -3968
5883 200
2389 8091
9658 6627
6248 8373
-8415 3004
8262 -5589
-9526 7593
-8878 -6069
2239 -2968
-1329 9491
-1441 4792
-6347 9144
5954 3801
7238 3005
8640 2067
705 5700
95 -4186
7999 666
7916 -9217
1977 -5515
-5961 -796
-6243 -6404
-7768 -7947
7041 -4054
-7795 -4039
-9449 1941
-3527 -4565
9571 -520
5937 -6404
3246 8918
-8119 -8011
-2773 -9392
4349 -3423
7520 4389
-5253 -4575
-6981 -3147
8259 3481
-4297 -4557
-4665 -6621
-4086 8657
-9004 -6667
-7602 -9643
3460 8890
9079 -5541
8236 9783
-6338 8147
-9229 6128
-9565 -7172
8923 4801
-5584 -5863
958 1886
-7590 8914
5132 32
2835 -9454
-2582 -155
5942 4786
-7470 -8893
-3169 -4148
5027 3630
-610 -1640
2843 3631
-631 -3941
2464 -9271
-6503 1949
-595 -6299
2086 5569
5606 1739
2866 -6690
7670 4386
7772 5311
-5681 -2388
9905 -5546
2456 1158
9051 6965
4401 8649
-4946 3776
9861 8864
6209 -4484
1495 -7994
-9621 9991
-4442 8900
3624 2431
-8683 8501
9081 -937
-5962 4063
-7147 6030
-693 7113
-1106 -3598
8685 3694
-2399 5082
-1385 5164
1885 -287
-2614 3967
2879 -1123
-2772 -1738
8175 3606
-8625 3697
-2097 -6819
-4181 -3447
-2788 -6758
4663 1102
8404 1496
4497 9987
5396 -8312
3721 -4508
-6368 -3478
-4621 7137
-3049 -2530
-4809 5792
7616 -8948
6657 510
-6733 -7518
8887 -9969
-2480 812
-4281 4471
7375 -2822
-970 -7074
-3382 -8642
-2410 -7952
-588 -6772
-5465 -4976
-4672 3000
-7020 -6855
-7003 -2912
2210 4487
1088 -3082
6491 -2710
8949 5046
-1831 -1803
233 4912
3435 8475
-3444 -9348
5404 -9628
-5082 -603
-5991 -8685
4368 -4229
2488 6453
-8632 -2819
-3998 -355
7074 9374
-4641 2641
9104 -4129
9571 3750
3079 -9385
-6034 9926
-8000 8138
-3970 -6919
-1566 1836
-4316 5963
359 4884
-773 7631
-7008 8581
4323 -6868
-1443 2836
2329 -4220
-9000 1710
-6127 7852
-8153 -1826
-7638 -8694
3908 1574
1465 -7083
-5393 -77
-2120 4510
-4004 9818
766 -6237
-587 4882
-2564 3192
-6895 -8150
-159 -8634
6444 3276
6541 -360
-9018 -8779
4917 863
-5840 4242
5747 -6202
9197 1517
8837 1286
3410 7242
-5725 4382
3903 -8465
-7940 -4812
-3679 6777
-1698 -211
-6511 1425
-6599 5407
5960 -3424
7206 5196
9785 -7223
-6402 -6378
-7900 1920
-3049 3467
8383 2964
1159 -603
-7675 -8164
-3860 7806
1112 -5732
-9570 -4807
837 5330
807 -8058
-3553 4992
-1801 -8749
-2106 989
795 4107
-4841 -13
8373 7670
6732 9102
2073 4513
2556 4832
4734 1189
4662 -7684
1613 864
8770 -4613
-560 1247
5672 -3913
531 -389
6308 -6410
-7779 -6824
8650 3407
-3332 -1675
6488 -534
3997 -1120
8510 7705
5314 -5464
-3547 -5244
6988 -4620
3143 9763
-7181 -9189
8680 3726
1081 4885
-5530 5277
48 287
-2568 880
2506 -5644
8642 9855
1457 8686
-8143 -9407
1559 6545
-9164 8590
8864 6790
-8245 4000
4553 -3927
-971 5641
-2685 -1145
-4200 364
-3572 9492
1577 -3513
7443 -6896
4014 -3413
-2021 4226
-8648 4066
-4058 5061
-7988 8048
