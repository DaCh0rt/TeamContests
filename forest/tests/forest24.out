81494.95 446061543.28
